torchvision/prototype/transforms/_augment.py

```diff
@@ -38,8 +38,7 @@ def __init__(
         self.value = value
 
     def _get_params(self, sample: Any) -> Dict[str, Any]:
-        image = query_image(sample)
-        img_c, img_h, img_w = get_image_dimensions(image)
+        img_c, img_h, img_w = query_image_dimensions(sample)
```

Comment:
Nit: It might be worth renaming this to to indicate that the method can be used not only for images but also videos. Spatial dims could be an option, though channels are not usually considered part of the spatial dims.

Reply:
We agreed offline to `{query, get}_chw` since it is concise, factual, and does not "bias" the name towards images.